docs/docs/03-interactivity-and-dynamic-uis.md

```diff
@@ -81,4 +81,4 @@ A common pattern is to create several stateless components that just render data
 
 * **Computed data:** Don't worry about precomputing values based on state — it's easier to ensure that your UI is consistent if you do all computation within `render()`. For example, if you have an array of list items in state and you want to render the count as a string, simply render `this.state.listItems.length + ' list items'` in your `render()` method rather than storing it on state.
 * **React components:** Build them in `render()` based on underlying props and state.
-* **Duplicated data from props:** Try to use props as the source of truth where possible. One valid use to store props in state is to be able to know its previous values, because props can change over time.
+* **Duplicated data from props:** Try to use props as the source of truth where possible. One valid use to store props in state is to be able to know its previous values, because props may change as a component is re-rendered.
```

Comment:
This seems to imply that props change as a result of the render function being called.  Maybe "... may change when a parent component is re-rendered" or "... may change as a result of a parent component re-rendering"?